Sources/Rownd/Views/HubWebView/HubWebViewController.swift

```diff
@@ -252,11 +252,13 @@ extension HubWebViewController: WKScriptMessageHandler, WKNavigationDelegate {
                 }
                 
             case .triggerSignInWithApple:
+                guard case .triggerSignInWithApple(let signInWithAppleMessage) = hubMessage.payload else { return }
```

Comment:
If no intent was initially passed, wouldn't payload be `nil` and thus nothing will happen when the corresponding button is pressed?

Reply:
Perhaps this could be simplified, but maybe something along these lines?

```swift
var signInWithAppleMessage: MessagePayload.TriggerSignInWithAppleMessage? = nil
if case .triggerSignInWithApple(let message) = hubMessage.payload {
    signInWithAppleMessage = message
}

self.hubViewController?.hide()
Rownd.requestSignIn(with: .appleId, signInOptions: RowndSignInOptions(intent: signInWithAppleMessage?.intent))
```

Reply:
When the intent is passed from the hub as undefined, the guard check still seems to be working.

![Screen Shot 2023-01-31 at 8 22 30 AM](https://user-images.githubusercontent.com/53702681/215772062-b7a4d492-361c-4f84-b1cf-36a129f1a4e8.png)